cmd/lukso/flags.go

```diff
@@ -458,9 +486,13 @@ func prepareValidatorStartFlags(ctx *cli.Context) (startFlags []string) {
 func preparePrysmStartFlags(ctx *cli.Context) (startFlags []string) {
 	startFlags = clientDependencies[prysmDependencyName].PassStartFlags(ctx)
 	startFlags = append(startFlags, prepareLogfileFlag(ctx.String(logFolderFlag), prysmDependencyName))
+	startFlags = append(startFlags, "--accept-terms-of-use")
```

Comment:
Don't accept TOS for user if user didn't accept. I dont know if you ask user for this - but this may put some heat on us, needlessly.

Reply:
User actually accepted TOS way earlier in the process of creating his working folder, so I think it's fine. What do you think @frozeman?

Reply:
good find @DubbaThony 
but the user accepted the terms already when installing the client. We will add a comment to that section to make sure its clear for people looking at the code.

Reply:
Done